lib/index.js

```diff
@@ -317,6 +317,7 @@ function writeRequireJS (version, destination, minify, callback) {
             if (base === 'Gruntfile') { return; }
             var relativePath = path.relative(famousPath, path.dirname(file));
             var module = path.join(namespace, relativePath, base);
+            module = module.split('\\').join('/');
```

Comment:
It probably should, but the core of the issue looks to be in the requirejs optimizer.  If `path.sep` is  used, everything is still broken on windows.